pages/common/codecommit.md

```diff
@@ -0,0 +1,20 @@
+# codecommit
```

Comment:
the page name `codecommit` should match the command name `aws-codecommit`, this page isn't valid.

Reply:
@kbdharun  I have removed the codecommit page  from the PR you can check now.